core/core/initutils.go

```diff
@@ -240,7 +240,7 @@ func getDownloadReleasedPolicy(ctx context.Context, downloadReleasedPolicy *gett
 func getDefaultFrameworksPaths() []string {
 	fwPaths := []string{}
 	for i := range getter.NativeFrameworks {
-		fwPaths = append(fwPaths, getter.GetDefaultPath(getter.NativeFrameworks[i]))
+		fwPaths = append(fwPaths, getter.GetDefaultPath(getter.NativeFrameworks[i]+".json"))
```

Comment:
maybe you can add a comment explaining that frameworks are downloaded with the extension
BTW is that true on Windows too?

Reply:
Not just the frameworks, everything is downloaded with the extension `.json`. I had to add ".json" because GetDefaultPath expects a filename and not the framework's name.

I will try it to run it on windows

Reply:
I confirm, that the issue exists on windows (windows 10), but this PR fixes it on all OS (tested on macos aswell)